hack/move-upstream.sh

```diff
@@ -35,7 +37,14 @@ if [[ -n "${1-}" ]]; then
 fi
 
 echo "++ Generating patch for ${selector} onto ${lastrev} ..." 2>&1
-git diff -p --raw --relative=Godeps/_workspace/src/${repo}/ "${selector}" -- Godeps/_workspace/src/${repo}/ > "${patch}"
+index=0
+for commit in $(git log --no-merges --format="%H" --reverse "${selector}" -- "Godeps/_workspace/src/${repo}/"); do
+  git format-patch --raw --start-number=${index} --relative="Godeps/_workspace/src/${repo}/" "${commit}^..${commit}" -o "${patch}"
+  let index+=10
```

Comment:
It's not terribly usual, but that's one way to `+=` in Bash:
```
$ a=1
$ a+=1
$ echo $a
11
$ let a+=1
$ echo $a
12
```

I personally write:
```
$ a=1
$ a=$(( $a + 1 ))
```
but that's my opinion what is more readable.